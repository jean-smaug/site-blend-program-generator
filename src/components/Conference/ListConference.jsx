import React from 'react';
import PropType from 'prop-types';

import Conference from './Conference';

const ListConference = ({ day }) => {
  const { eight, ten, fourteen, sixteen } = day;

  return (
    <div>
      <Conference {...eight[0]} />
      <Conference {...ten[0]} />
      <Conference {...fourteen[0]} />
      <Conference {...sixteen[0]} />
    </div>
  );
};
ListConference.propTypes = {
  day: PropType.shape({
    eight: Conference.propTypes,
    ten: Conference.propTypes,
    fourteen: Conference.propTypes,
    sixteen: Conference.propTypes,
  }).isRequired,
};

export default ListConference;
