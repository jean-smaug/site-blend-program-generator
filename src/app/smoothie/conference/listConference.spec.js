import React from 'react';
import { shallow } from 'enzyme';
import ListConference from './listConference.component';

const props = {
  day: {
    eight: [],
    ten: [],
    fourteen: [],
    sixteen: [],
  },
};

describe('listConference.component', () => {
  it('should render listConference.component', () => {
    const listConference = shallow(<ListConference {...props} />);
    expect(listConference).toMatchSnapshot();
  });
});
