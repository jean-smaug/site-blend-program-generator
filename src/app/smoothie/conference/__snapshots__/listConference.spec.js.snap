// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`listConference.component should render listConference.component 1`] = `
Array [
  <div
    className="conferences"
>
    <Conference
        conferences={Array []}
        timeBegin={8}
        timeEnd={10}
    />
    <Conference
        conferences={Array []}
        timeBegin={10}
        timeEnd={12}
    />
    <Conference
        conferences={Array []}
        timeBegin={14}
        timeEnd={16}
    />
    <Conference
        conferences={Array []}
        timeBegin={16}
        timeEnd={18}
    />
</div>,
]
`;
